### PR TITLE
Fix #1060: Docs. Header links

### LINKED ISSUE #1060: Language changes does not affect to header links
* **What is the expected behavior?**
image

* **What is the current behavior?**
image

### PR DESCRIPTION
**Pull request by Lemonka93**
Fixes #1060

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

**Comment by asvae** [comment]
@Derranion can you take a look at this one?

File: packages/docs/src/components/header/Header.vue [hunk 1]
```diff
@@ -57,33 +67,27 @@ const PropsMixin = Vue.with(Props)
...
-        },
-        {
-          text: this.$t('menu.contribution'),
-          icon: 'fa fa-share-alt',
-          to: `/${(this as any).$root.$i18n.locale}/contribution/documentation-page`,
-        },
-      ],
-    }
+  get links () {
+    return [
```
Comment (m0ksem) [inline packages/docs/src/components/header/Header.vue:71]:
```suggestion
	const locale = this.$root?.$i18n.locale

    return [
```

File: packages/docs/src/components/header/Header.vue [hunk 2]
```diff
@@ -57,33 +67,27 @@ const PropsMixin = Vue.with(Props)
...
+      {
+        text: this.$t('menu.overview'),
+        icon: 'fa fa-eye',
+        to: `/${(this as any).$root.$i18n.locale}/introduction/overview`,
```
Comment (m0ksem) [inline packages/docs/src/components/header/Header.vue:75]:
```suggestion
        to: `/${locale}/introduction/overview`,
```

File: packages/docs/src/components/header/Header.vue [hunk 3]
```diff
@@ -57,33 +67,27 @@ const PropsMixin = Vue.with(Props)
...
+        icon: 'fa fa-github',
+        url: 'https://github.com/epicmaxco/vuestic-ui',
+        target: '_blank',
+      },
+      {
+        text: this.$t('menu.contribution'),
+        icon: 'fa fa-share-alt',
+        to: `/${
+          (this as any).$root.$i18n.locale
+        }/contribution/documentation-page`,
```
Comment (m0ksem) [inline packages/docs/src/components/header/Header.vue:88]:
```suggestion
        to: `/${locale}/contribution/documentation-page`,
```

**Review by m0ksem — APPROVED** [review]
A couple of simplifications can be made.

**Pull request merged**